andela/storm-backend
# #168094076 moving models folder to src

#### What this PR does
This PR move models folder to src folder to enable Regle tracking

#### Tasks completed
- rename eslint config file
- move models folder to src folder

#### How this can be manually tested
- Clone this branch
- Change directory into it
- Check the src folder for the models folder

#### What are the relevant pivotal tracker stories?
[#168094076](https://www.pivotaltracker.com/story/show/168094076)
